certified-connectors/Starmind/apiProperties.json

```diff
@@ -1,5 +1,117 @@
 {
   "properties": {
+    "connectionParameterSets": {
+      "uiDefinition": {
+        "displayName": "Authentication Type",
+        "description": "Type of authentication to be used."
+      },
+      "values": [
+        {
+          "name": "starmind-apikey",
+          "uiDefinition": {
+            "displayName": "Authenticate with API key (technical user)",
+            "description": "Use a Starmind-issued API key of a technical user to access the Starmind API.",
+            "tooltip": "Authenticate using an API key belonging to a technical Starmind user for secure, automated integrations."
+          },
+          "parameters": {
+            "sm_BaseUrl": {
+              "type": "string",
+              "uiDefinition": {
+                "displayName": "The Starmind network. Example: yourcompany.starmind.com",
+                "description": "Specify your Starmind network. Example: yourcompany.starmind.com",
+                "tooltip": "e.g., yourcompany.starmind.com",
+                "constraints": {
+                  "required": "true",
+                  "tabIndex": 2
+                }
+              }
+            },
+            "api_key": {
+              "type": "securestring",
+              "uiDefinition": {
+                "displayName": "API key as provided by Starmind",
+                "description": "Enter the API key as provided by Starmind",
+                "tooltip": "Use an API key issued by Starmind for secure, automated access as a technical user.",
+                "constraints": {
+                  "clearText": false,
+                  "required": "true",
+                  "tabIndex": 3
+                }
+              }
+            }
+          }
+        },
+        {
+          "name": "starmind-oauth",
+          "uiDefinition": {
+            "displayName": "Authenticate with your Starmind user (OAuth 2.0)",
+            "description": "Sign in using your own Starmind user account via OAuth 2.0.",
+            "tooltip": "Authenticate via OAuth 2.0 with your personal Starmind user instead of an API key."
+          },
+          "parameters": {
+            "token": {
+              "uiDefinition": {
+                "constraints": {
```

Comment:
this seems required, otherwise the Power Automate UI will fail with JS issue accessing undefined: